libraries/chain/block_log.cpp

```diff
@@ -303,9 +303,9 @@ namespace eosio { namespace chain {
       uint64_t      last_block_position() const { return read_buffer<uint64_t>(data() + size() - sizeof(uint64_t)); }
       chain_id_type chain_id() const { return preamble.chain_id(); }
 
-      fc::optional<genesis_state> get_genesis_state() const {
-         return std::visit(overloaded{[](const chain_id_type&) { return fc::optional<genesis_state>{}; },
-                                      [](const genesis_state& state) { return fc::optional<genesis_state>{state}; }},
+      std::optional<genesis_state> get_genesis_state() const {
+         return std::visit(overloaded{[](const chain_id_type&) { return std::optional<genesis_state>(); },
+                                      [](const genesis_state& state) { return std::optional<genesis_state>(state); }},
```

Comment:
Any reason to move away from uniform initialization here?

Reply:
I ran across a couple of places that looked like using `std::make_optional` was appropriate and I got overzealous with that change.  This is a remnant of that, so I'll definitely roll this back.